.eslintrc

```diff
@@ -2,66 +2,28 @@
   "parser": "babel-eslint",
   "parserOptions": {
     "sourceType": "module",
+    "ecmaVersion": 6,
     "allowImportExportEverywhere": true
   },
-  "extends": "airbnb",
+  "extends": [
+    "formidable/configurations/es6-react",
+    "prettier"
+  ],
   "env": {
     "browser": true,
-    "node": true
+    "node": true,
+    "commonjs": true,
+    "es6": true
   },
   "rules": {
+    "filenames/match-regex": "off",
```

Comment:
Can you ticket out doing this in the future? We won't do `dash-case` like the default is, but I think we _can_ come up with a workable, consistent regex.